tests/gemm/test_low_latency_blockscaled_gemm.py

```
@@ -0,0 +1,325 @@
+        torch.einsum(
+            "mkl,nkl->mnl",
+            _decode_ab_to_f32(a) * _reference_scales(sfa, sf_vec_size, mnkl[2]),
+            _decode_ab_to_f32(b) * _reference_scales(sfb, sf_vec_size, mnkl[2]),
+        )
+        * alpha.cpu()
+        + bias.cpu().to(torch.float32)[:, None, None]
+    ).to(torch.bfloat16)
+
+
+def _make_runner():
+    return _cute_dsl_low_latency_blockscaled_gemm_runner(100, False)
```

Ruff (0.16.3)

[error] Undefined name `_cute_dsl_low_latency_blockscaled_gemm_runner`

(F821)